ui/src/linking/linking-api.ts

```diff
@@ -71,15 +88,15 @@ export const getSuggestedContinuousLocationTracks = async (
         locationTrackPointUpdateType: locationTrackEndPoint.updateType,
         bbox: bboxString(bbox),
     });
-    const uri = linkingUri('location-tracks', 'suggested');
+    const uri = linkingUri(undefined, 'location-tracks', 'suggested');
```

Comment:
Tätä frontin kontekstimallia voisi ehkä vielä harkita. Nyt siis frontin kannalta layoutcontextin branch on vain design id, joka on tietenkin null/undefined main branchin tapauksessa. Se tarkoittaa että tällaisissa käyttöpaikoissa (joissa ei ole mietlä sanoa kontekstia vaan vain branch) joudutaan "main" kommunikoimaan näin, sen sijaan että sanottaisi esim. `MAIN` jostain vakiosta/enum-arvosta.

Tuota ei kannata tähän jätte-pullariin enää sotkea, mutta asia voisi varmaan olla paremmin.

Reply:
`undefined`:issa on se hyvä puoli, että branchissa oleminen voidaan tarkastaa todella helpolla tyyliin `branchId ? teeBranchiJuttu() : teeEiBranchiJuttu()`. Heikentää tosin luettavuutta näissä merkittävästi, vaikka IDEA auttaakin näyttämällä parametrien nimet. Mutta eipä johonkin vakioon vertailukaan kauheasti koodausmukavuuden kannalta ikävämpi juttu olisi, varsinkin kun sen voi funktioida

Reply:
Toisena juttuna, onkos näiden fronttisidonnoista jo oma tikettinsä? Reduxista näille löytyy jo paikka, joten sidonta olisi periaatteessa mahdollista jo nyt (vaikka se kannattaa ihan omana hommanaan tehdäkin)

Reply:
Ei ole tikettiä vielä. Teenpä sellaisen.

Reply:
https://extranet.vayla.fi/jira/browse/GVT-2609